llvm/test/Transforms/VectorCombine/RISCV/vector-interleave2-splat-e64.ll

```diff
@@ -0,0 +1,17 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -mtriple=riscv64 -mattr=+v,+m,+zvfh %s -passes=vector-combine | FileCheck %s
+; RUN: opt -S -mtriple=riscv32 -mattr=+v,+m,+zvfh %s -passes=vector-combine | FileCheck %s
+
+; We should not form a i128 vector.
+
+define void @interleave2_const_splat_nxv8i64(ptr %dst) {
```

Comment:
Does this have some bad interaction with zve32x that required a separate test?

Reply:
> Does this have some bad interaction with zve32x that required a separate test file?

Correct, from what I'd tried zve32x doesn't like SEW=64 in general.